### ISSUE TITLE
influxdbr is not abandoned anymore

### ISSUE THREAD

**Issue by dleutnant** [mvadu/influxdbr2#8]
Hi mvadu, 

Since I am actively maintaining the influxdbr package again, could you please update your README.md to indicate the difference between the forks (users seem to be confused because of the "2" in the repo name, which misleadingly suggests a newer version)?

Thanks!
Cheers
Dominik

**Comment by mvadu** [comment]
@dleutnant agreed.. I think I forked this when you were off (when my PR was pending for more than 6 months without any response). I have few functions which kind of helped to query Influx (with XTS objects). I will probably push them as a PR to your main branch and close this fork. Since you have now published in CRAN i see no reason to keep this up.

**Comment by dleutnant** [comment]
Ok, thanks. Looking forward to your PR's (please use the dev branch)!
